### PR TITLE
[fr] follow last changes

### PR DESCRIPTION
**Pull request by antograssiot**
ref #3614, #3615, #3617, #3620

### CODE REVIEW

File: fr/development/errors.rst [hunk 1]
```diff
@@ -166,13 +166,43 @@ les méthodes HTTP.
...
+
+    .. versionadded:: 3.1.7 ConflictException a été ajoutée.
+
+.. php:exception:: GoneException
+
+    Utilisée pour faire une erreur 410 Gone.
+
+    .. versionadded:: 3.1.7 GoneException a été ajoutée.
+
+Pour de détails sur les codes de statut d'erreur HTTP 4xx, regardez
```
Comment (cake17) [inline fr/development/errors.rst]: Pour **plus** de détails ?

File: fr/development/errors.rst [hunk 2]
```diff
@@ -166,13 +166,43 @@ les méthodes HTTP.
...
-    Utilisée pour faire une Erreur 501 Non Implémentée.
+    Utilisée pour faire une erreur 501 Non Implémentée.
+
+.. php:exception:: ServiceUnavailableException
+
+    Utilisée pour faire une erreur 503 Service Unavailable.
+
+    .. versionadded:: 3.1.7 Service Unavailable a été ajoutée.
+
+Pour de détails sur les codes de statut d'erreur HTTP 5xx, regardez
```
Comment (cake17) [inline fr/development/errors.rst]: here too : **plus** is missing

**Comment by cake17** [comment]
@antograssiot nice work :+1:

**Comment by antograssiot** [comment]
@cake17 done

**Pull request merged**